lib/workers/global/index.ts

```diff
@@ -133,10 +134,13 @@ export async function start(): Promise<number> {
     await instrument('config', async () => {
       // read global config from file, env and cli args
       config = await getGlobalConfig();
-      if (config?.globalExtends) {
+      if (is.nonEmptyArray(config?.globalExtends)) {
```

Comment:
why this change?

Reply:
because for some reason it is an empty array even when not explicitly defined, this if condition is basically always truthy.
this has been my experience when debugging this section at least.